gate_controller/worker.py

```
@@ -343,18 +383,32 @@ def enqueue(item):
+        "max_candidates": max_burst_candidates,
+    }
+    collector = BurstCollector(enqueue, **collector_options)
+    sampler = None
+    augmentation_collector = None
+    sampling_enabled = bool(snapshot_sampling is not None and snapshot_sampling.enabled)
+    if snapshot_sampling is not None:
+        if sampling_enabled:
+            augmentation_collector = BurstCollector(enqueue, **collector_options)
+            add_snapshot_candidate = augmentation_collector.add
+        else:
+            add_snapshot_candidate = lambda *_: None
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `add_snapshot_candidate` as a `def`

(E731)